src/test.py

```diff
@@ -21,18 +21,18 @@ def test_with_image(img_path):
 
     if left_eye_pred == 0 and right_eye_pred == 0:
         score += 1
-        cv2.putText(img, "Closed", (10, height-20), font, 1, (255, 255, 255), 1, cv2.LINE_AA)
+        cv2.putText(img, "Asleep", (10, height-20), font, 1, (255, 255, 255), 1, cv2.LINE_AA)
```

Comment:
This was already resovled in PR, did you pull before sending this??

Reply:
It was done on the test .py not on main.py

Reply:
![Screenshot from 2020-05-14 11-22-10](https://user-images.githubusercontent.com/40380481/81923394-83cc5300-95d5-11ea-87f4-3f4913faadb8.png)

Reply:
Thanks for clarifying that. I didn't notice it.